sympy/stats/crv_types.py

```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
 
     return rv(name, LogLogisticDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+#Logit-Normal distribution------------------------------------------------------
+
+class LogitNormalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 's')
+    set = Interval(0, 1)
+
+    @staticmethod
+    def check(mu, s):
+        _value_check(s**2 > 0, "Scale parameter s must be positive.")
+
+    def logit(self, x):
```

Comment:
Shouldn't it be private?
```suggestion
    def _logit(self, x):
```

Reply:
Yeah I've made the required changes in the latest commit